### ISSUE TITLE
Strange noise instead of alarm sound

### ISSUE THREAD

**Issue by LubomirKovalcik** [NightscoutFoundation/xDrip#1564]
### Subject of the issue
When phone is in sleep, and alarm for low or high glucose activates with default alarm sound, it sometimes plays strange "electronic noise" instead of actual alarm sound. 

### Your environment
- Version: e14bae1-2020.07.13
- Data source: xDrip+ Sync Follower
- Is your issue specific to a device like and Android phone or a smartwatch? Xiaomi Redmi 9 + Android 10 + MIUI 11

### Expected behavior
Plays alarm sound. 

### Actual behavior
Sometimes plays "electric noise" instead of alarm sound

### Steps to reproduce the behavior:
For example:
1. Create alarm for low/high BG
2. Put phone to sleep and wait for alarm
3. Once phone has woken and alarm is being played, there is either correct sound or electric noise

**Comment by tolot27** [comment]
Wow, what a great example of filling a bug report correctly. Thanks! :smile:

Do you observe this behavior with system alarms, too? Does it happen with different alarm sounds?

**Comment by Navid200** [comment]
I have only one of my alarms that misbehaves sometimes.  It seems to me that when it misbehaves, it only plays a portion of the sound clip.  It is not clear to me why it only happens to one alarm.  This is an alarm that uses a personal notification sound that I have added to my system sounds on the phone.

I have no idea if this is a related issue to what is described here or not.

**Comment by Navid200** [comment]
@LubomirKovalcik Are you still experiencing this?
If yes, by any chance, are you using the ascending volume profile?

**Comment by LubomirKovalcik** [comment]
> Wow, what a great example of filling a bug report correctly. Thanks! 😄
> 
> Do you observe this behavior with system alarms, too? Does it happen with different alarm sounds?

No system sounds works correctly (like regular notifications or alarm).


> @LubomirKovalcik Are you still experiencing this?
> If yes, by any chance, are you using the ascending volume profile?

Yes I am still experiencing this, but only on Xiaomi Redmi 9 (Android 10 + MIUI 11). 
I also have follower settings on Samsung A70, and main collector is Mi9 (MIUI 11) and there is no such a problem.

I am not sure about ascending volume profile. I will have to check it somehow, but not quite sure where, because I could not find it anywhere in settings.  

**I will try to set custom alarm sound in xDrip and see if thats make any change. I will report back.**

**Comment by Navid200** [comment]
You can choose your volume profile:
Settings -> Alarms and Alerts -> Glucose Alerts Settings -> Alert Volume Profile

**Comment by Navid200** [comment]
If we figure out what is common between the sound files that cause this, we may be able to identify a characteristic (size, name, format, ...) that is the problem and provide guidance to users how to avoid the issue.

Thanks for your work

**Comment by Navid200** [comment]
@LubomirKovalcik Any update on this?

**Comment by Navid200** [comment]
Closing due to inactivity

**Comment by gilko86** [comment]
I recently started hearing the strange 'electronic noise' too, it was driving me nuts. I turned every possible notification off but I _think_ I have found the solution. It only started since installing AndroidAPS. There is a setting called **Import Sounds** in the Settings - > Inter-app settings menu. I switched this off and haven't heard it since. 

Looking at the adb shell output just after hearing the noise helped identify where to look. 

`logcat | toybox grep -i "callingPack" | toybox sed -e "s/$/\n/"
`

_01-25 15:21:22.142   978  1997 D PackageManager: setEnabledSetting : userId = 0 packageName = info.nightscout.androidaps cmp = androidx.work.impl.background.systemalarm.RescheduleReceiver newState = 1 callingPackage = 10473/**info.nightscout.androidaps**_

Fingers crossed.
